src/meshweb/static/admin/iframe_check.js

```diff
@@ -0,0 +1,62 @@
+async function checkIframed() {
+  // Check if we're in an iframe
+  const inIframe = window.self !== window.top;
+  
+  // If we are, do nothing
+  if (inIframe) {
```

Comment:
FWIW I think you should handle mobile by just adding an `|| isMobile()` here, so they don't deal with any of the iframe stuff and have just a pure HTML experience with no map